mkdocs.yml

```diff
@@ -33,6 +33,7 @@ markdown_extensions:
   - pymdownx.magiclink
   - pymdownx.smartsymbols
   - pymdownx.superfences
+  - pymdownx.tilde
```

Comment:
necessary for `~~foo~~` to render foo in strikeout, which we do on our TLS configuration history page